lib/streamlit/elements/map.py

```diff
@@ -13,14 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""A Python wrapper around DeckGl."""
```

Comment:
Add an empty line below this.

Also, this pydoc isn't 100% correct. This would be more correct: `A wrapper for simple PyDeck scatter charts`